third_party/jsign/index.mjs

```diff
@@ -15,9 +15,9 @@
 import {spawn} from 'node:child_process';
 import {resolve} from 'node:path';
 
-import {downloadHttpsFile} from '../../src/build/download_file.mjs';
-import {getFileChecksum} from '../../src/build/get_file_checksum.mjs';
-import {getRootDir} from '../../src/build/get_root_dir.mjs';
+import {downloadHttpsFile} from '../../client/src/build/download_file.mjs';
```

Comment:
This dependency is broken. The jsign library shouldn't depend on the client.
Keep `build/` where it is and update the client to point to the common code instead.